sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultClientTest.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@EnabledIfEnvironmentVariable(named = "AZURE_KEYVAULT_CERTIFICATE_NAME", matches = ".*")
+public class KeyVaultClientTest {
+    private KeyVaultClient keyVaultClient;
+    private String certificateName;
+
+    @BeforeEach
+    public void setEnvironmentProperty() {
+        keyVaultClient = new KeyVaultClient(
+            System.getenv("AZURE_KEYVAULT_URI"),
+            System.getenv("AZURE_KEYVAULT_TENANT_ID"),
+            System.getenv("AZURE_KEYVAULT_CLIENT_ID"),
+            System.getenv("AZURE_KEYVAULT_CLIENT_SECRET"));
+        certificateName = System.getenv("AZURE_KEYVAULT_CERTIFICATE_NAME");
+    }
+
+    @Test
+    public void testGetAliases() {
+        assertNotNull(keyVaultClient.getAliases());
```

Comment:
please check if aliases are fetched as expected, do not use assertNotNull.
And see if below 2 tests could be updated, too.